src/app/modules/item/pages/item-details/item-details.component.spec.ts

```diff
@@ -18,8 +17,7 @@ describe('ItemDetailsComponent', () => {
       set: {
         providers: [
           { provide: CurrentContentService, useValue: {
-            setCurrent: (_i: NavItem) => {},
-            setPageInfo: (_p: any) => {},
+            setCurrent: (_i: any) => {},
```

Comment:
should `_i` be of type `ContentInfo` ?

Reply:
For tests, we do not care (especially as we do not do anything with it).